testing/netatalk/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Contributor:
+# Maintainer: Alexander Rigbo <alex@dnb.nu>
+pkgname=netatalk
+pkgver="3.1.10"
+pkgrel=0
+pkgdesc="Netatalk is a freely-available Open Source AFP fileserver."
+url="http://netatalk.sourceforge.net/"
+arch="all"
+license="GNU GPLv2"
+depends="db libgcrypt acl"
```

Comment:
These dependencies are autodetected via dynamic linking, so it’s not needed (nor desirable) to list them here.